src/devices/boiler.cpp

```diff
@@ -46,7 +46,7 @@ Boiler::Boiler(uint8_t device_type, int8_t device_id, uint8_t product_id, const
     }
 
     // register values for master boiler/cascade module
-    reserve_telgram_functions(25); // reserve some space for the telegram registries, to avoid memory fragmentation
+    reserve_telgram_functions(26); // reserve some space for the telegram registries, to avoid memory fragmentation
```

Comment:
I think we can remove this line. There is so much free mem left on the ESP32 now